### ISSUE TITLE
Problem with spot numbering

### ISSUE THREAD

**Issue by omerishaq** [omerishaq/BMC-Annotation#1]
During the testing I found a bug in the tool regarding the spot numbering which only occurs if a person when annotating an image say 'A' decide to leave the annotation midstream (without completing annotation of image A) and load another image 'B'. The application is fine if the annotation of the current image which is loaded is completed and then another image is loaded. It is also fine if the annotation of the image 'A' is left midstream but the application is restarted before annotation image 'B'

In other words if you are annotating an image 'A' and want to annotate an image 'B' without completing the annotation of 'A', either complete 'A' or restart application before starting 'B'.
